### PR TITLE
Update headers from 2018 to 2019 once for all.

### PR DESCRIPTION
**Pull request by He-Pin**
Happy new year.

### CODE REVIEW

**Comment by hepin1989** [comment]
Separate commit for easy reviewing, and can do a squash before getting merged.

File: project/CopyrightHeader.scala [hunk 1]
```diff
@@ -45,7 +48,7 @@ trait CopyrightHeader extends AutoPlugin {
   )
 
   val CurrentYear = java.time.Year.now.getValue.toString
-  val CopyrightPattern = "Copyright \\([Cc]\\) (\\d{4}(-\\d{4})?) (Lightbend|Typesafe) Inc. <.*>".r
+  val CopyrightPattern = "Copyright \\([Cc]\\) (\\d{4}([-–]\\d{4})?) (Lightbend|Typesafe) Inc. <.*>".r
```
[the remark this answers is not in the document]
Comment (raboof) [reply project/CopyrightHeader.scala]: I guess we want to pick one, though?

[the remark this answers is not in the document]
Comment (raboof) [reply project/CopyrightHeader.scala]: OK for now, though

**Comment by raboof** [comment]
PLS BUILD

File: project/CopyrightHeader.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (hepin1989) [reply project/CopyrightHeader.scala]: But they are currently both in the source code ,wich will by passing the matching.

**Review by raboof — APPROVED** [review]
Thanks!

**Review by johanandren — APPROVED** [review]

**Comment by johanandren** [comment]
CLA is signed by @hepin1989, not sure what the CLA checker is up to.

**Pull request merged**